commands/command_migrate_import.go

```diff
@@ -29,6 +30,67 @@ func migrateImportCommand(cmd *cobra.Command, args []string) {
 	}
 	defer db.Close()
 
+	if migrateNoRewrite {
+		if len(args) == 0 {
+			ExitWithError(errors.Errorf("fatal: the migrate import command requires a list of files to import when using the --no-rewrite flag"))
```

Comment:
Great; I think dying here is the right thing to do. Let's shorten this message a little bit while we're here. For instance, the phrase "the migrate import command" might be unnecessary, since this will be the first line of output after a user runs `git lfs migrate import`.

What about: "fatal: expected one or more file(s) with --no-rewrite"?